modin/pandas/test/test_groupby.py

```diff
@@ -2165,3 +2165,22 @@ def test_groupby_with_frozenlist():
     pandas_df = pandas_df.set_index(["a", "b"])
     modin_df = from_pandas(pandas_df)
     eval_general(modin_df, pandas_df, lambda df: df.groupby(df.index.names).count())
+
+
+@pytest.mark.parametrize(
+    "by_func",
+    [
+        lambda df: df.columns[0],
+        lambda df: list(df.columns[:-1]),
+        lambda df: [df.columns[0], df.iloc[:, 1].copy()],
```

Comment:
Isn't column 1 out of bounds in the `iloc`?

Reply:
Column `df.iloc[:, 1]` is the same thing as `df["timestamp1"]`. I will use `df["timestamp1"]` to make it clearer.